mtp_api/apps/security/tests/utils.py

```diff
@@ -82,34 +97,74 @@ def _get_credit_values(credit_filters, sender_profile_id, prisoner_profile_id, p
     )
 
 
+def _get_prisoner_profile(
+    j: int, filter_set: dict, number_of_prisoners_to_use: int, fake_prisoner_names: dict, fiu
+) -> Tuple[Optional[PrisonerProfile], str]:
```

Comment:
shouldn't the typing hint be `Tuple[Optional[…], Optional[str]]` for these methods?

Reply:
or even more strictly `Union[Tuple[…, str], Tuple[None, None]]`

Reply:
I've made the first change, I don't know if we gain much with the second but i'm happy to implement if you're keen